### PR TITLE
Nix flake

### PR DESCRIPTION
**Pull request by jbgi**

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    haskell-nix.url = "github:input-output-hk/haskell.nix";
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-20.09";
+    utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, utils, haskell-nix, ... }:
+    (utils.lib.eachSystem [ "x86_64-linux" "x86_64-darwin" ] (system:
+      let
+        legacyPackages = import ./nix {
+          inherit sources system nixpkgs iohkNix;
```
Comment (johnalotoski) [inline flake.nix:14]: Cyclic dependency between legacyPackages and sources?

Comment (jbgi) [reply flake.nix:14]: no, lazy evaluation make it ok.

**Review by disassembler — APPROVED** [review]

**Review by Jimbo4350 — APPROVED** [review]

**Pull request merged**